### DISCUSSION TITLE
Do you use TriliumNext?

### DISCUSSION THREAD

**Discussion by CobriMediaJulien** [mabeyj/trilium-collection-views#55] [General]
Hey mabeyj,

do you use this with TriliumNext or what note takign app are you using right now?

**Comment by mabeyj** [comment]
Hey, yes, I do use TriliumNext, but I haven't upgraded past 0.90.4 yet. (I sync with an Arch Linux server and there's no newer server packages available yet.)

Been using it for a few months and haven't noticed any issues.
